tests/test_memory_core/test_memory_core.py

```diff
@@ -0,0 +1,365 @@
+from memory_core import memory_core_genesis2
+from memory_core.memory_core import gen_memory_core, Mode
+import glob
+import os
+import tempfile
+import shutil
+import fault
+import random
+from hwtypes import BitVector
+from gemstone.common.testers import ResetTester, ConfigurationTester
+from memory_core.memory_core_magma import MemCore
+import magma
+import canal
+import coreir
```

Comment:
again.

Reply:
the first one is necessary but others are stray, fixed in newest commit